faq.md

```diff
@@ -5,64 +5,150 @@ layout: faq
+
+If your organization (or consortia) has its own Galaxy server, then you may be able to run tutorials on that.  This can be ideal because then the instance you are teaching on is the same you your participants will be using after the training.  They'll also be able to revisit any analysis they did during the training.  If you pursue this option you'll need to work with your organization's Galaxy Admins to confirm that
+
+- the server can support a room full of people all doing the same analysis at the same time.
+- all tools and reference datasets needed in the tutorial are locally installed.  Some tutorials include scripts that install all needed tools and data on an existing server.
+- all participants will be able to create/use accounts on the system.
+
+TODO: Training Infrastructure as a Service on useGalaxy.eu
+
+Finally, you can also run your training on cloud-based infrastructures.  Galaxy has been available on Amazon Web Services for years, and is also available on many national research infrastructures such as Jetstream (United States), GenAP (Canada), GVL (Australia), CLIMB (United Kingdom), and so on.
+
+## What are the best practices for teaching with Galaxy?
 
-- Parse the website to find interesting tutorials to teach
-- Find a Galaxy instance to use for the workshop
-    - Check the annotated Galaxy instance
-    - Training Infrastructure as a Service on useGalaxy.eu
-    - Docker image
-    - Deployment of the tools, data, etc on an exisiting instance using our provided scripts
 - Have a look at our [Good practices slides]()
 
-# How can I make sure a tutorial is running on a Galaxy instance?
 
-- Badge?
-- Installations
+## How do I get help?
+
+The support channel for instructors is the same as for individual learners.  We suggest you start by posting a question to the Galaxy Training Network [Gitter chat](https://gitter.im/Galaxy-Training-Network/Lobby).  Anyone can view the discussion, but you'll need to login (using your GitHub or Twitter account) to add to the discussion.
+
+If you have questions about Galaxy in general (that are not training-centric) then there are [several support options](https://galaxyproject.org/support/).
+
+
+# Contributing
+
+First off, thanks for your interest in contributing to the Galaxy training materials!
+
+Individual learners and instructors can make these training more effective by contributing back to them. You can report mistakes and errors, create more content, etc. Whatever is your background, there is a way to contribute.  See the [contributing page]({{ site.github_repository }}/CONTRIBUTING.md)) for how.
 
-# How can I contribute?
+TODO: @tnabtaf suggests moving rest of this section to CONTRIBUTING.md
```

Comment:
I like the idea of having one place (FAQ) to centralize all information or redirection. On the other hand, the `CONTRIBUTING.md` is the one linked by GitHub when a PR is open.
@shiltemann @bgruening what do you think?

Reply:
I like having the rest of section here as well, especially since it is just a couple sentences pointing people in the right direction.

Reply:
We discussed recently a picture of the 3 pillars of contribution, maybe we can include such a image here and link to the contributing.md page?